docs/ides/metals.md

```diff
@@ -4,28 +4,33 @@ title: Metals
 sidebar_label: Metals
 ---
 
-Metals is a work-in-progress language server for Scala that supports a diverse number of text
-editors such as Visual Studio Code, `vim`, Sublime Text and Atom.
+Metals is a language server for Scala that supports a diverse number of text
+editors such as Visual Studio Code, Vim, Emacs, Eclipse, Sublime Text and Atom.
 
 |                              | Build import | Compile        | Test                  | Run                  | Debug                 |
 | ---------------------------- | ------------ | -------------- | --------------------- | -------------------- | --------------------- |
-| **Metals**                   | ✅           | ✅             | ✅                   | ✅                   | ✅                   |
+| **Metals**                   | ✅           | ✅             | ✅                    | ✅                   | ✅                    |
 
-At the moment, the BSP integration only supports build import and compilation (with compiler
-diagnostics integrated in the editor). This functionality is enough to provide a fluent developer
-experience. Support for test is planned.
+
+The BSP integration in metals supports build import, compilation, Test, Run, and
+Debug functionality.
```

Comment:
Mainly wanted to just update this to match what the graph above is showing since this hasn't been update since 2018 it look like.